src/ir_Argo.cpp

```diff
@@ -0,0 +1,282 @@
+/*
+Node MCU/ESP8266 Sketch to emulate Argo Ulisse 13 DCI remote
+Copyright 2017 Schmolders
+*/
+
+#include "ir_Argo.h"
+#include <algorithm>
+#include "IRremoteESP8266.h"
+#include "IRutils.h"
+#include <Arduino.h>
+
+
+//  ARgo
+
+// Constants
+// using SPACE modulation. MARK is always const 400u
+#define ARGO_PREAMBLE_1           6400U  // Mark
+#define ARGO_PREAMBLE_2           3300U  // Space
+#define ARGO_MARK                  400U
+#define ARGO_ONE_SPACE            2200U
+#define ARGO_ZERO_SPACE            900U
+
+#if SEND_ARGO
+// Send a Argo A/C message.
+//
+// Args:
+//   data: An array of ARGO_COMMAND_LENGTH bytes containing the IR command.
+//
+// Status: STABLE
```

Comment:
BETA at best. ;-) We probably need others to confirm it's working before we can count it as stable. Or have some unit tests for it.